com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/MtaDescriptorMerger.java

```diff
@@ -44,7 +44,11 @@ public DeploymentDescriptor merge(DeploymentDescriptor deploymentDescriptor, Lis
         handlerFactory.getPlatformMerger(platform)
                       .mergeInto(mergedDescriptor);
 
-        deploymentDescriptor = mergedDescriptor;
+        DeploymentDescriptor validatedDescriptor = handlerFactory.getDescriptorParametersCompatabilityValidator(mergedDescriptor,
```

Comment:
Can it go under DescriptorValidator.validate()

Reply:
Why? In HandlerFactory exist method which return DescriptorParametersValidator which is similar to getDescriptorParametersCompatabilityValidator